src/libpriv/rpmostree-core.c

```diff
@@ -2484,8 +2468,7 @@ run_pre_sync (RpmOstreeContext *self,
   if (!get_package_metainfo (self, path, &hdr, NULL, error))
     return FALSE;
 
-  if (!rpmostree_pre_run_sync (pkg, hdr, self->ignore_scripts,
-                               rootfs_dfd, cancellable, error))
+  if (!rpmostree_pre_run_sync (pkg, hdr, rootfs_dfd, cancellable, error))
```

Comment:
not sure if this would matter, but may be split into two lines like earlier implementation?